src/pretix/control/forms/renderers.py

```diff
@@ -21,90 +21,16 @@
 #
 from bootstrap3.renderers import FieldRenderer, InlineFieldRenderer
 from bootstrap3.text import text_value
-from django.forms import CheckboxInput, CheckboxSelectMultiple, RadioSelect
 from django.forms.utils import flatatt
 from django.utils.html import format_html
 from django.utils.safestring import mark_safe
 from django.utils.translation import pgettext
-from i18nfield.forms import I18nFormField
 
+from pretix.base.forms.renderers import PretixFieldRenderer
 
-def render_label(content, label_for=None, label_class=None, label_title='', label_id='', optional=False):
-    """
-    Render a label with content
-    """
-    attrs = {}
-    if label_for:
-        attrs['for'] = label_for
-    if label_class:
-        attrs['class'] = label_class
-    if label_title:
-        attrs['title'] = label_title
-    if label_id:
-        attrs['id'] = label_id
 
-    if text_value(content) == '&#160;':
-        # Empty label, e.g. checkbox
-        attrs.setdefault('class', '')
-        attrs['class'] += ' label-empty'
-
-    builder = '<{tag}{attrs}>{content}{opt}</{tag}>'
-    return format_html(
-        builder,
-        tag='label',
-        attrs=mark_safe(flatatt(attrs)) if attrs else '',
-        opt=mark_safe('<br><span class="optional">{}</span>'.format(pgettext('form', 'Optional'))) if optional else '',
```

Comment:
We need to find a way to overwrite how fields are marked as optional in presale vs. control. You removed the text „Optional“ from below a label in control.